src/java.desktop/share/classes/javax/imageio/ImageReader.java

```diff
@@ -2876,4 +2876,11 @@ protected static void checkReadParamBandSettings(ImageReadParam param,
         // Create a new image based on the type specifier
         return imageType.createBufferedImage(destWidth, destHeight);
     }
+
+    /**
+     * @implNote The default implementation of this method invokes {@link #dispose()}.
+     */
+    public void close() {
```

Comment:
(Not a reviewer) I think we need to add the `@since` tag to javadoc.

Reply:
Good catch! I added `@since` in https://github.com/openjdk/jdk/pull/12098/commits/7991ea30f18a7e98f4f37cae40c8f86dba6ea7c9.